completions/bash/kpod

```diff
@@ -273,6 +916,277 @@ _kpod_rename() {
     esac
 }
 
+_kpod_container_run() {
+	local options_with_args="
+		--add-host
+		--attach -a
+		--blkio-weight
+		--blkio-weight-device
+		--cap-add
+		--cap-drop
+		--cgroup-parent
+		--cidfile
+		--cpu-period
+		--cpu-quota
+		--cpu-rt-period
+		--cpu-rt-runtime
+		--cpuset-cpus
+		--cpus
+		--cpuset-mems
+		--cpu-shares -c
+		--device
+		--device-read-bps
+		--device-read-iops
+		--device-write-bps
+		--device-write-iops
+		--dns
+		--dns-option
+		--dns-search
+		--entrypoint
+		--env -e
+		--env-file
+		--expose
+		--group-add
+		--hostname -h
+		--init-path
```

Comment:
I don't see init-path in flags, should this be interactive (which seems to be missing)?

Reply:
oops, never mind interactive, that's a boolean.